### PR TITLE
Fix #214: Update Docker installation instructions for Windows users

### LINKED ISSUE #214: Update Docker wiki regarding the use of WSL terminal for the windows users
Since Docker in Windows requires WSL for executing its commands (refer here: https://docs.docker.com/desktop/install/windows-install/#system-requirements). We should clearly state that WSL is also an pre-requisite for the Window Users and every `make` command in Docker Setup needs to be executed within the WSL terminal.

### PR DESCRIPTION
**Pull request by gp201**
This PR fixes #214 .

This PR updates the Docker installation instructions for Windows users to install and use WSL2.

### CODE REVIEW

**Comment by gp201** [comment]
@seanlip PTAL

File: Installing-Oppia-using-Docker.md [hunk 1]
```diff
@@ -74,7 +74,9 @@ To install Oppia under Docker, follow these steps:
...
-2. **[For Windows users]**: You need to have `make` installed on your system. Confirm whether `make` is installed on your system by running `make --version`. If `make` is not installed, you can install it by following the instructions provided in the following link: [Installing make on Windows](https://stackoverflow.com/questions/32127524/how-to-install-and-use-make-in-windows).
+2. **For Windows users**: Prior to proceeding with this guide, make sure you have activated the WSL2 backend for Windows in Docker Desktop. You can find instructions on how to do this [here](https://docs.docker.com/desktop/wsl/). Additionally, ensure that you execute all the commands present in this guide within the Windows Subsystem for Linux 2 (WSL2) terminal.
+
+You need to enable the WSL 2 backend for Docker Desktop. Refer to the [official documentation](https://docs.docker.com/docker-for-windows/wsl/) for more information.
```
Comment (seanlip) [inline Installing-Oppia-using-Docker.md]: @gp201 This part of the doc is outside point 2 and is a general thing that's not just for Windows users. Should point 2 be split into two sub-bullet points after the "For Windows users" intro, and should this sentence be placed in the second bullet point?

**Review by seanlip — CHANGES_REQUESTED** [review]
Thanks @gp201, one comment.

File: Installing-Oppia-using-Docker.md (hunk 1 above)
Comment (gp201) [reply Installing-Oppia-using-Docker.md]: I have combined the 2 points.

**Comment by gp201** [comment]
@seanlip PTAL

**Review by seanlip — APPROVED** [review]
Thanks! LGTM, merging.

**Pull request merged**